hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestBackupUtils.java

```diff
@@ -67,8 +68,11 @@ public Path run() {
     Path bulkOutputDir = ugi.doAs(new PrivilegedAction<Path>() {
       @Override
       public Path run() {
-        try {
-          return BackupUtils.getBulkOutputDir("test", conf, false);
+        try (FileSystem fs = FileSystem.get(conf)) {
+          String tmp = conf.get(HConstants.TEMPORARY_FS_DIRECTORY_KEY,
```

Comment:
we should probably make this a util method somewhere rather than duplicate it in all the tests... it could be in BackupUtils or maybe CommonFSUtils